netrd/reconstruction/correlation_matrix.py

```diff
@@ -47,11 +47,12 @@ def fit(self, TS, num_eigs=10, quantile=0.9):
         vals = vals[idx]
         vecs = vecs[:, idx]
 
-        # construct the precision matrix
+        # construct the precision matrix and store it
```

Comment:
It might be a good idea to enforce the documentation of everything that is stored in self.results. The docstring of each fit/dist method should precisely describe everything that it puts in self.results. For example, we could add it to the template.